app/kubemci/pkg/gcp/urlmap/urlmapsyncer.go

```diff
@@ -70,14 +72,21 @@ func (s *URLMapSyncer) EnsureURLMap(lbName string, ing *v1beta1.Ingress, beMap b
 	if err == nil {
 		fmt.Println("url map", name, "exists already. Checking if it matches our desired url map", name)
 		glog.V(5).Infof("Existing url map: %v\n, desired url map: %v", existingUM, desiredUM)
+		// TODO(G-Harmon): Figure out how to properly calculate the fp. Using Sha256 returned a googleapi error.
+		desiredUM.Fingerprint = existingUM.Fingerprint
 		// URL Map with that name exists already. Check if it matches what we want.
 		if urlMapMatches(desiredUM, existingUM) {
 			// Nothing to do. Desired url map exists already.
 			fmt.Println("Desired url map exists already")
 			return existingUM.SelfLink, nil
 		}
-		// TODO: Require explicit permission from user before doing this.
-		return s.updateURLMap(desiredUM)
+		if forceUpdate {
+			fmt.Println("Updating URL Map because it's different.")
```

Comment:
Use consistent message for all resources.
For comparison, this is the message for health checks:
`fmt.Println("Updating existing health check", name, "to match the desired state")`

Reply:
done. Good idea.